notebooks/239-freevc/239-freevc.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            wav_src = torch.from_numpy(wav_src).unsqueeze(0)
</pre><p>compiled_model accepts numpy too, so it is no need to convert into torch tensor</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>